Mathlib/Algebra/Order/Ring/Unbundled/Basic.lean

```diff
@@ -751,8 +751,6 @@ lemma max_mul_mul_le_max_mul_max [PosMulMono α] [MulPosMono α] (b c : α) (ha
     mul_le_mul (le_max_right a c) (le_max_right b d) hd (le_trans ha (le_max_left a c))
   max_le (by simpa [mul_comm, max_comm] using ba) (by simpa [mul_comm, max_comm] using cd)
 
-/-- Binary **arithmetic mean-geometric mean inequality** (aka AM-GM inequality) for linearly ordered
-commutative semirings. -/
 lemma two_mul_le_add_sq [ExistsAddOfLE α] [MulPosStrictMono α]
```

Comment:
I still think that's a sensible docstring, no?

Reply:
I moved it to `four_mul_le_add_sq` since I that's how the (squared and division-free) [AM GM inequality](https://en.wikipedia.org/wiki/AM–GM_inequality) is actually stated?

Reply:
I'm not married to moving the docstring, but I think it fit's `four_mul_le_add_sq` better than `two_mul_le_add_sq`. We can also put it for both.

Reply:
I think it's better to have it for both